integration_test/pg/storage_test.exs

```diff
@@ -33,19 +27,10 @@ defmodule Ecto.Integration.StorageTest do
     assert Postgres.storage_up(correct_params) == {:error, :already_up}
   end
 
-  test "storage up (wrong credentials)" do
-    refute Postgres.storage_up(wrong_user) == :ok
```

Comment:
Incorrect credentials cause an exit, which I'm not sure should be caught.

```
20:51:08.074 [error] GenServer #PID<0.229.0> terminating
** (stop) %Postgrex.Error{message: nil, postgres: %{code: :invalid_authorization_specification, file: "miscinit.c", line: "362", message: "role \"randomuser\"
 does not exist", pg_code: "28000", routine: "InitializeSessionUserId", severity: "FATAL"}}
```

Any advice here?